### PR TITLE
Delete a Wishlist item feature

### PR DESCRIPTION
**Pull request by MDShahedurRahman**
I have added codes for Delete a Wishlist item. This has been tested and confirmed.

### CODE REVIEW

File: tests/test_routes.py [hunk 1]
```diff
@@ -260,6 +260,39 @@ def test_query_wishlist_list_by_customer_id(self):
         for wishlist in data:
             self.assertEqual(wishlist["customer_id"], test_customer_id)
 
+    def test_delete_wishlist_item(self):
+        """It should Delete a Wishlist Item"""
+        test_wishlist = self._create_wishlists(1)[0]
+        item = ItemFactory()
+        item.wishlist_id = test_wishlist.id
+
+        req = {"customer_id": test_wishlist.customer_id}
```
Comment (neerajkanuri) [inline tests/test_routes.py:269]: Check if we're using this in the code ?

Comment (MDShahedurRahman) [reply tests/test_routes.py:269]: Took those lines from "def test_get_wishlist_items(self):" Line: 129 - 134. I thought the test will be similar for get an item and delete an item. Please advise me on how to proceed.

Comment (neerajkanuri) [reply tests/test_routes.py:269]: Makes sense.

**Review by neerajkanuri — APPROVED** [review]
LGTM

**Comment by MDShahedurRahman** [comment]
Merging delete a Wishlist item feature to the main repo.

**Pull request merged**